stdlib/public/runtime/CMakeLists.txt

```diff
@@ -103,8 +102,7 @@ add_swift_library(section_magic_end OBJECT_LIBRARY IS_STDLIB IS_STDLIB_CORE
 
 set(object_target_list)
 foreach(sdk ${SWIFT_CONFIGURED_SDKS})
-  is_elfish_sdk("${sdk}" IS_ELFISH)
-  if(IS_ELFISH)
+  if("${SWIFT_SDK_${sdk}_OBJECT_FORMAT}" STREQUAL "ELF")
```

Comment:
Because that would be smart and why I had put EFLISH_SDKS into the top level scope.  I knew I had forgotten some part of the change.  Thanks for pointing that out.